README.md

```diff
@@ -397,3 +397,195 @@ const UserForm = () => (
   </div>
 );
 ```
+
+&nbsp;
+
+---
+
+&nbsp;
+
+## FAQ
+
+> ### When should I use `createKeyedResource` vs `createSingleEntryResource`?
+
+The the main difference between `createKeyedResource` and `createSingleEntryResource` is how the created `resource` stores its entries. Internally, every `resource` organizes the data it receives from the completed load function into entries. A `resource`'s main concern is how the data in the entries should be available to your app, not necessarily how the external api is structured or called.
+
+- `createKeyedResource` creates a resource with multiple entries that are organized by key. [See the first parameter on how entries are keyed](https://github.com/chadwatson/react-remote-resource#createkeyedresource).
+
+- `createSingleEntryResource` creates a resource with only a single entry. All data, regardless of the structure, will be stored in only one entry.
+
+Here are some examples scenerios that show when each of the :
+
+#### Example Users API:
+
+Assuming You have a users api that takes an id and returns user information:
+
+```javascript
+/*
+  
+  /api/users/:id
+
+  Example Response:
+  {
+    name: "name",
+    id: {userId},
+    ...
+  }
+
+*/
+```
+
+**Scenerio 1:**
+
+The app only needs the current users information. Since the app only one entry from the api (the current users information and no others), `createSingleEntryResource` would work well.
+
+```jsx
+const load = (id) => fetch(`/api/users/${id}`);
+const userResource = createSingleEntryResource(load);
+
+const currentUserId = ...;
+
+const AboutMe = () => {
+  const [user] = userResource.useEntry(currentUserId);
+
+  return ...;
+}
+```
+
+**Scenerio 2**
+
+The app needs multiple users from the API. Since the app needs multiple entries from the api organized by key then createKeyedResource would then be a better choice.
+
+```jsx
+const load = (id) => fetch(`/api/users/${id}`);
+const usersResource = createKeyedResource(id => id, load);
+
+const User = ({id}) => {
+  const [user] = usersResource.useEntry(id);
+
+  return ...;
+}
+
+const UserList = ({ids}) => {
+  return ids.map(id => <User key={id} id={id} />
+}
+```
+
+### Example Clients API:
+
+Assuming You have a clients api that takes an account_rep_id and returns a list of clients for that specific account rep:
+
+```javascript
+/*
+
+/api/clients/:account_rep_id
+
+{
+  123454: {
+      ...
+  },
+  508923: {
+      ...
+  },
+  14: {
+      ...
+  },
+  995: {
+      ...
+  }
+}
```

Comment:
Since the description above says the API returns a "list of clients" should the data be more shaped like this?

```javascript
[
  {
    id: 123454,
    ...
  },
  {
    id: 508923,
    ...
  },
  {
    id: 14,
    ...
  },
  {
    id: 995,
    ...
  }
]
```

Reply:
Sounds good!